src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexNode.cs

```diff
@@ -2086,5 +2085,75 @@ internal SymbolicRegexNode<TSet> PruneAnchors(uint prevKind, bool contWithWL, bo
                     return this;
             }
         }
+
+        /// <summary>Represents a list of transitions without duplicate effect arrays</summary>
```

Comment:
This comment could include that it keeps the list of transitions unique, expects higher priority transitions to be added first and ignores equivalent lower priority transitions.

Reply:
I'll update the comment.